sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -655,6 +651,10 @@ export interface RetryOptions {
     timeoutInMs?: number;
 }
 
+export { RheaAmqpMessageHeader }
+
+export { RheaAmqpMessageProperties }
```

Comment:
Why do we need to re-export things from rhea-promise?

Reply:
The intention was to retain the things that were being exported before.

Reply:
We don't need to.. actually!
I'll remove.

Reply:
Removed